locale/uk_UA/common.xml

```diff
@@ -99,30 +141,47 @@
 	<message key="common.error.framesRequired"><![CDATA[Ця сторінка потребує підтримки фреймів браузером. Для переходу до версії сторінки без фреймів <a href="{$url}">натисніть сюди</a>.]]></message>
 	<message key="common.event">Подія</message>
 	<message key="common.export">Експорт</message>
+	<message key="common.finish">Завершено</message>
 	<message key="common.fullscreen">Повноекранний режим</message>
 	<message key="common.fullscreenOff">Відключення повноекранного режиму</message>
-	<message key="common.false">False</message>
+	<message key="common.false">Хиба</message>
 	<message key="common.file">Файл</message>
+	<message key="common.file.namingPattern">{$username}, {$userGroup}, {$originalFilename}</message>
```

Comment:
This accidentally reverts necessary changes: https://github.com/pkp/pkp-lib/commit/cca21c12ea53f80a213e496a1dd82ca77f580a65#diff-3b5ac1283ce98563f3403a121de3d2e8L110

Is this translation based on OJS 3.1.0-1? If so, it may be necessary to re-apply some changes. You can see these by running (e.g. in `lib/pkp` for this repository):

    git diff -b -u ojs-3_1_0-1..master locale/uk_UA

Reply:
Quite possible that already translated tags were taken from earlier release. But I don't see any other problems apart from already mentioned. I will look at the translation consistency in compare with English version.

Reply:
I would suggest using `git diff` to identify possible conflicts:

    git diff ojs-3_1_0-1..master locale/uk_UA